test/unit/index.js

```diff
@@ -73,15 +73,19 @@ describe('minimal-cipher', function() {
           () => Math.floor(Math.random() * 255));
       }
 
-      async function encryptStream({data}) {
+      async function encryptStream({data, queSize = 5, chunkSize = 5}) {
```

Comment:
Should this be `queueSize`?

Reply:
you are correct here. I though `que` was a term for a line, it turns out I was completely wrong and `queue` is a data structure that is like a line, I made the correction here and good catch: https://github.com/digitalbazaar/minimal-cipher/pull/29/commits/5583046fb336464f46cb5ce7b5dd14de660d52ad